### PR TITLE
Downgrade ruby and bundler because heroku doesn't support it yet

### PR DESCRIPTION
**Pull request by xmunoz**
Still this: https://github.com/heroku/heroku-buildpack-ruby/issues/1273

Also, perform minor and patch package upgrades.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -228,9 +226,6 @@ GEM
       net-protocol
       timeout
     nio4r (2.5.8)
-    nokogiri (1.13.3)
-      mini_portile2 (~> 2.8.0)
-      racc (~> 1.4)
     nokogiri (1.13.3-x86_64-linux)
```
Comment (colinxfleming) [inline Gemfile.lock:229]: this fights back for me on a local env (mac) - I'm going to submit a commit that brings this back to the general instance

**Review by colinxfleming — APPROVED** [review]
looks good to me, other than nokogiri gems being bound to a system. non-native installs are annoying but kinda necessary here. hang tight, commit incoming

File: Gemfile.lock (hunk 1 above)
Comment (xmunoz) [reply Gemfile.lock]: nice catch!

File: Gemfile.lock [hunk 2]
```diff
@@ -190,6 +190,7 @@ GEM
     launchy (2.5.0)
       addressable (~> 2.7)
     libsqreen (1.0.4.0.0)
+    libv8-node (16.10.0.0)
     libv8-node (16.10.0.0-x86_64-linux)
```
Comment (xmunoz) [inline Gemfile.lock:194]: Maybe it's worth removing all linux-specific dependencies. I'll try to do that now.

**Comment by xmunoz** [comment]
jk, that doesn't work, removed last commit

**Pull request merged**